docs/dev_guide/newcomers.rst

```diff
@@ -150,30 +150,28 @@ Next we will want to setup the conda environment and we will need to add the `co
 .. code:: bash
 
     $ conda config --add channels conda-forge
-    $ conda create -n sunpy-dev sunpy
+    $ conda create -n sunpy-dev pip
```

Comment:
So the main change now is to install pip then pip install sunpy[dev] this way you get the full deps installed from pypi instead of half the deps from conda and then the ones we forgot in the recipe from pypi.